spec/controllers/presentations_controller_spec.rb

```diff
@@ -6,10 +6,19 @@
 
     before { mock_pundit_user_as(admin_person) }
 
-    describe 'GET #new' do
-      subject { get :new }
+    describe 'GET #edit' do
+      let(:presentation) { FactoryGirl.create(:presentation) }
+      subject { get :edit, params: { id: presentation } }
 
-      it { is_expected.to render_template :new }
+      it { is_expected.to render_template :edit }
+      it { is_expected.to be_successful }
+    end
+
+    describe 'GET #edit' do
```

Comment:
Fixing up spec after rebase mixup, and will be reorganizing per @brendandeere's comment above.